### ISSUE TITLE
Open Question - ability to use other fonts?

### ISSUE THREAD

**Issue by CaptainGrock** [AlekPet/ComfyUI_Custom_Nodes_AlekPet#25]
Hi - I didn't see any documentation, but is there a folder that your node is reading from to access other fonts than the default 5? If not, as a feature request, can you either have a separate font folder or the ability to access the Windows font folder (if implementing I'm guessing it's easier to use a subfolder of the custom node)?

Thanks so much in advance!

**Comment by AlekPet** [comment]
Yes, support for third-party fonts is possible, but not yet implemented, I just thought they weren’t particularly in demand.
